app/models/measure.rb

```diff
@@ -95,4 +115,10 @@ def additional_code_sort
       additional_code.code.to_s
     end
   end
+
+  private
+
+  def declarable_types
+    @declarable_types ||= YAML.load_file('config/declarable_types.yml')
+  end
```

Comment:
Maybe I'm wrong, but I think this will look it up every time the controller is reloaded (once per call).

Why not putting it into a custom configuration that will be loaded at startup, since this list will only change on deploy? (specifically using `Rails::Application.config_for` )

https://edgeguides.rubyonrails.org/configuring.html#custom-configuration

Reply:
Yep, we can do that!